### PR TITLE
LPS-149188

### PR DESCRIPTION
**Pull request by Seiphon**

### CODE REVIEW

**Comment by Seiphon** [comment]
ci:test:sf

**Comment by Seiphon** [comment]
ci:test:gradle-plugins

**Comment by Seiphon** [comment]
Hi @lawrence-lee , according to the last running of gradle-plugins test, there is only one test failure (smoke test), but there is no error message print, and it can be running successfully on my local machine. So I have no idea why does that happen. Can you give some help?

**Comment by Seiphon** [comment]
ci:test:sf

**Comment by Seiphon** [comment]
ci:test:gradle-plugins

**Comment by Seiphon** [comment]
ci:test:gradle-plugins

**Comment by Seiphon** [comment]
ci:test:gradle-plugins

**Comment by Seiphon** [comment]
ci:test:gradle-plugins

File: modules/sdk/gradle-plugins/src/main/java/com/liferay/gradle/plugins/NodeDefaultsPlugin.java [hunk 1]
```diff
@@ -190,9 +190,9 @@ private void _configureTaskPublishNodeModule(
 		}
 	}
 
-	private static final String _NODE_VERSION = "16.13.0";
```
Comment (lawrence-lee) [inline modules/sdk/gradle-plugins/src/main/java/com/liferay/gradle/plugins/NodeDefaultsPlugin.java:193]: @Seiphon , you can't change it like this. You should just add an override in the workspace plugin

**Comment by lawrence-lee** [comment]
@simonjhy, can you confirm if this is the correct place? 

https://github.com/liferay/liferay-portal/blob/master/modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/FrontendPlugin.java#L206-L210

**Comment by lawrence-lee** [comment]
ci:test:gradle-plugins

**Comment by lawrence-lee** [comment]
ci:test:gradle-plugins

**Pull request closed without merging**